core/src/main/java/de/danoeh/antennapod/core/util/playback/Playable.java

```diff
@@ -176,11 +173,18 @@
 
     void setChapters(List<Chapter> chapters);
 
+    /**
+     * Returns the location of the image or null if no image is available.
+     * The location can either be a URL or a local path.
+     */
+    String getImageLocation();
+
     /**
      * Provides utility methods for Playable objects.
      */
     class PlayableUtils {
-        private PlayableUtils(){}
+        private PlayableUtils() {
```

Comment:
Not the scope of your PR but I think this class should be in its own file.

Reply:
I can provide another PR after this PR is merged, to avoid merge conflicts.

Reply:
See PR #4962.